### DISCUSSION TITLE
How to get proper styling for form element

### DISCUSSION THREAD

**Discussion by AloysA** [squidfunk/mkdocs-material#2256] [Q&A]
As part of our product documentation we want to include some forms that, using some javascript, compute some values on the fly for hardware estimations.

However, I noticed that the form element are not include in the stylesheets. Can somebody help me to get the form related css important so it has the material look-and-feel? I did not get it to work using overrides, and adding the material css as extra css breaks the whole layout.

**Comment by squidfunk** [comment]
Yes, there are no default styles for form elements, except for buttons. Unfortunately, I don't know of anyone having done this for Material for MkDocs and it's not on the roadmap, but it should be perfectly achievable with some CSS.

**Reply by AloysA** [reply]
I've created some custom CSS for now. The actual Material styling for forms would have been nicer, but that was too complex to achieve (effort vs gain). Thanks for the theme though, we love it.
